client/src/commands/encointer_ceremonies.rs

```diff
@@ -772,7 +909,12 @@ async fn get_attendees_for_community_ceremony(
 	let mut attendees = Vec::new();
 	let mut noshows = Vec::new();
 	for storage_key in storage_keys.iter() {
-		match api.get_storage_by_key(storage_key.clone(), at_block).await.unwrap().unwrap() {
+		match api
+			.get_storage_by_key(storage_key.clone(), maybe_at)
+			.await
+			.unwrap_or(Some(Reputation::VerifiedLinked(0)))
```

Comment:
This default seems unintuitive for me, shouldn't it be unverified reputable?

Reply:
Surprisingly, it only fails in the case of  VerifiedLinked. It's hacky, but this at least preserves the main information of interest

Reply:
Ahhh, ok. I thought that we unwrap an option here, got it now. No, this makes perfect sense to me, actually. All other enum variants haven't changed, so decoding is not a problem. With the custom rpc though, there is just a very high change that there is a `VerifiedUnlinked` in one of the returned reputations, and unfortunately, this makes the whole rpc call fail.

Reply:
added a todo to remove the hack asap
https://github.com/encointer/encointer-node/issues/364